pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template

```diff
@@ -52,8 +52,10 @@ jobs:
           echo "::set-output name=branch::${head}"
 
           head="${head#refs/heads/}" # Strip "refs/heads/" prefix.
+          # If we have at least 2 '#' in ref
           if [[ $(echo "$head" | grep -o '#' | wc -l) -gt 1 ]]; then
-            base="$(echo $head | cut -d '#' -f 1)"
```

Comment:
As we previously used bash var substitution syntax 2 lines before, I thought it made sense to use it here also.